robots/EDUCATOR/gyro.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
```

Comment:
Maybe call this `square-gyro.py` since the non-gyro equivalent is `square.py`?

Reply:
Good idea.